test/ShallowWrapper-spec.jsx

```diff
@@ -1897,6 +1897,16 @@ describe('shallow', () => {
       expect(parents.at(1).hasClass('bar')).to.equal(true);
       expect(parents.at(2).hasClass('bax')).to.equal(true);
     });
+
+    it('should work with component', () => {
+      const Foo = React.createClass({
+        render() {
+          return <div className="bar" />;
+        },
+      });
+      const wrapper = shallow(<Foo />);
+      expect(wrapper.find('.bar').parent()).to.have.length(0);
```

Comment:
In this case, `.find('.bar')` *is* the root - I'm not even sure `.find('.bar')` will find anything. Could you add an assertion that `wrapper.find('.bar')` has length 1?